reportssql/indicator_functions.sql

```diff
@@ -913,6 +913,42 @@ WHERE
 END$$ 
 DELIMITER ;
 
+DROP FUNCTION IF EXISTS Testing_Indicator3a;
+
+DELIMITER $$
+CREATE FUNCTION Testing_Indicator3a(
+    p_startDate DATE,
+    p_endDate DATE,
+    p_startAgeInMonths INT(11),
+    p_endAgeInMonths INT (11),
+    p_includeStartAge TINYINT(1)) RETURNS INT(11)
+    DETERMINISTIC
+BEGIN
+    DECLARE result INT(11) DEFAULT 0;
+    DECLARE dateOfVirologicHIVTest DATE;
+
+    SET dateOfVirologicHIVTest = getDateOfVirologicTest(p_patientId, p_startDate, p_endDate);
+
+SELECT
+    COUNT(DISTINCT pat.patient_id) INTO result
+FROM
+    patient pat
+WHERE
+    patientAgeAtReportEndDateIsBetween(pat.patient_id, p_endDate, p_startAgeInMonths, p_endAgeInMonths) AND
```

Comment:
@tmvumbi2 I don't think this check is necessary, and it's also wrong, because for dissagregation we should check the age at test date (as it's done later on in patientAgeAtVirologicHIVTestIsBetween) and not at the report end date.

I think we should remove this check and delete the function.